internal/ddt/node.go

```diff
@@ -346,6 +346,21 @@ func (n *Node) Version() string {
 func (n *Node) Asset(name string) (*NodeAsset, error) {
 	path := filepath.Join(n.Path, name)
 
+	assets, err := n.Assets()
+
+	for _, a := range assets {
+
```

Comment:
You can remove this empty line, I think when running `go fmt` it'd remove it, too.